scripts/populate_preprint_providers_notification_subscriptions.py

```diff
@@ -0,0 +1,20 @@
+from website.app import setup_django
+setup_django()
+from osf.models import NotificationSubscription, PreprintProvider
+from api.preprint_providers.permissions import GroupHelper
+
+
+def main():
```

Comment:
Shouldn't this be part of the migration? (h/t mattf)

Reply:
It is part of the migration 0084. However, because creating fake preprint providers is not a migration but through a script which is ran after the migration is finished, developers might want to run this script so that fake preprint providers have NotificationSubscription instances attached to them. So this is a script solely for local dev usage.
On second thought, this could probably go into the `populate_fake_preprint_providers` script.

Reply:
adding it to populate fake preprint providers makes sense I think